tests/tiup/lib.sh

```diff
@@ -261,3 +261,14 @@ function run_dmctl_with_retry() {
 	cat $dmctl_log
 	exit 1
 }
+
+function ensure_start_relay() {
+	# manually enable relay for source1 after v2.0.2
+	if [[ "$PRE_VER" != "v2.0.0" ]] && [[ "$PRE_VER" != "v2.0.1" ]]; then
+		dmctl_log="get-worker.txt"
+		# always use CUR_VER, because we might use tiup mirror in previous steps.
+		tiup dmctl:$CUR_VER --master-addr=master1:8261 operate-source show -s mysql-replica-01 >$dmctl_log 2>&1
+		worker=$(grep "worker" $dmctl_log | awk -F'"' '{ print $4 }')
+		run_dmctl_with_retry $CUR_VER "start-relay -s mysql-replica-01 $worker" "\"result\": true" 1
```

Comment:
Not use retry?

Reply:
the first time should succeed. I just want to reuse existing function to do a result checking.